RefrigeratorSaraFarkash/RefrigeratorSaraFarkash/CItem.cs

```diff
@@ -0,0 +1,124 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Claims;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace RefrigeratorSaraFarkash
```

Comment:
Please note that properties create the private field automatically when they are simple getters and setters so some of your code is unnecessary.
Please read: https://learn.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/auto-implemented-properties
To learn more